lib/ansible/module_utils/compat/paramiko.py

```diff
@@ -10,5 +10,8 @@
 paramiko = None
 try:
     import paramiko
-except (ImportError, AttributeError) as err:  # paramiko and gssapi are incompatible and raise AttributeError not ImportError
+# paramiko and gssapi are incompatible and raise AttributeError not ImportError
+# When running in FIPS mode, cryptography raises InternalError
+# https://bugzilla.redhat.com/show_bug.cgi?id=1778939
+except (ImportError, AttributeError, Exception) as err:
```

Comment:
Just nuke the other exceptions, and only leave `Exception`. You can keep documentation of the individual exceptions known things raise.

Reply:
😊